docs/references/uvh5_memo.tex

```diff
@@ -493,20 +567,70 @@ \subsubsection{Rank-3 Array Convention}
 window-axis), UVH5 files should be written such that they conform to Version
 1.0.
 
+\subsection{Version 1.1}
+Historically, only a single phase center was supported and only sidereal or
+unprojected (zenith drift without w projection) phasing types were supported.
+When multiple phase center phasing was added, along with support for more
+types of phase centers, the  \textbf{phase\_center\_catalog},
+\textbf{phase\_center\_id\_array}, \textbf{phase\_center\_app\_ra},
+\textbf{phase\_center\_app\_dec}, and \textbf{phase\_center\_frame\_pa}
```

Comment:
this honestly looks really nasty in the rendered documet. Maybe we can make it an itemize instead of a comma separated sentence?

Reply:
good idea